compute_endpoint/PyPI.md

```diff
@@ -0,0 +1,7 @@
+# Globus Compute Endpoint
+
+[Globus Compute](https://globus-compute.readthedocs.io/en/latest/) is a distributed Function as a Service (FaaS) platform that enables flexible, scalable, and high performance remote function execution. Unlike centralized FaaS platforms, Globus Compute allows users to execute functions on heterogeneous remote computers, from laptops to campus clusters, clouds, and supercomputers.
+
+This package provides the functionality for running and managing [endpoints](https://globus-compute.readthedocs.io/en/latest/endpoints.html), which are the services that manage the execution of functions on target machines.
+
+If you want to submit functions to your own endpoints or to endpoints managed by someone else, you'll need the companion [Globus Compute SDK](https://pypi.org/project/globus-compute-sdk/) package.
```

Comment:
"You will need the companion [...] (https://) package to submit functions to endpoints, whether your own or someone else's."